scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/syntax/SCollectionSyntax.scala

```diff
@@ -57,7 +57,7 @@ final class PredictSCollectionOps[T](private val self: SCollection[T]) {
    * @param signatureName
    *   name of [[org.tensorflow.framework.SignatureDef]] s to be used to run the prediction.
    */
-  def predict[V: Coder, W](
+  def predict[V: Coder](
```

Comment:
This is a breaking change. would you like to mark this one for 0.13 ?

Reply:
Yeah let's. The parameter is unused so not sure why it lived on, maybe for backwards compatibility?